packages/exam/index.ts

```
@@ -0,0 +1,415 @@
+                const result = await cursor.sort({ score: -1 }).toArray();
+                return result.map((tsdoc: any, index: number) => [index + 1, tsdoc]);
+            },
+            applyProjection: (_: any, rdoc: any) => rdoc,
+        };
+        console.log('[Exam Plugin] 核心逻辑已加固');
+    });
+
+    // i18n: 简单加载中/英文词条
+    ctx.i18n.load('zh', {
+        'Exam': '考试',
+        'Exams': '考试',
```

Biome (2.4.12)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)